specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/network.json

```diff
@@ -143,6 +143,21 @@
       "description": "Reference to another subresource.",
       "x-ms-azure-resource": true
     },
+    "ChildResource": {
```

Comment:
I think all child resources also have `name` and `etag` properties

Reply:
Removed ChildResource type.